packages/core/src/components/TextArea/Proofreader/index.tsx

```diff
@@ -690,13 +695,26 @@ export class Proofreader extends React.Component<Props & WithStylesProps, State,
 
 export default withStyles(
   theme => {
-    const { input } = buildInputStyles(theme);
+    const { input, input_important: inputImportant } = buildInputStyles(theme);
     const { unit } = theme;
 
     // Add space for controls
     const inputPadding = unit * 2; // pattern.regularButton horizontal padding
     const paddingBottom = inputPadding + unit * 4;
 
+    const { backgroundColor: colorImportant } = inputImportant;
+
+    const controlsAfter = {
+      content: '""',
```

Comment:
I don't think you need to duplicate these properties, since important will just inherit from the base styles.